8-mcp/1-mcp/README.md

```diff
@@ -0,0 +1,17 @@
+# Using tools with an MCP server from NPM with vanilla LangChain4j
+
+## Requirements for running this app
+By default this app uses OpenAI's gpt-4o model.
+
+You need to set the `OPENAI_API_KEY` environment variable to the value of your OpenAI key.
+
+## Running this app
+1. Run `./mvnw clean quarkus:dev`
+2. See the output in the console
+3. Check that a file `playground/sum.py` was created with a Python script that sums two numbers
+4. When done, hit the `q` key in the Quarkus console
+
+## Running with local Ollama instead
+If you'd like to run with a local Ollama instance instead, first you need to ensure a local Ollama instance is running on port `11434` and the `llama3.2` model is pulled.
```

Comment:
`application.properties` shows `qwen2.5-coder:latest` yet this shows `llama3.2`. Which is the correct one?

Reply:
Hmm, I'll retry running the test a few times with both models and see which one works better. I remember having serious issues with both though.

Reply:
Yeah I find the same thing too. Nothing is a good substitute for something you pay for :/